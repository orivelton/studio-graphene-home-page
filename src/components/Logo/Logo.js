import React from 'react';
import './Logo.scss';

const Logo = () => <a href="/" className="logo uppercase">Logo</a>;

export default Logo;
