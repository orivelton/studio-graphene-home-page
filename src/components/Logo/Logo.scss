@import '../../assets/scss/helper.scss';

.logo {
  padding: em(28) em(40);
}
