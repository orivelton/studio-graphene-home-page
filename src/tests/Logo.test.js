import React from 'react';
import renderer from 'react-test-renderer';
import Logo from '../components/Logo';

test('renders correctly Logo', () => {
  const tree = renderer.create(<Logo />).toJSON();
  expect(tree).toMatchSnapshot();
});
