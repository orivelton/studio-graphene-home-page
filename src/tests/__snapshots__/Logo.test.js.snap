// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Logo 1`] = `
<a
  className="logo uppercase"
  href="/"
>
  Logo
</a>
`;
